### DISCUSSION TITLE
Need help for fixing the font style

### DISCUSSION THREAD

**Discussion by Uppi-Sempai** [community/community#141608] [Pull Requests] [answered]
### Select Topic Area

Question

### Body

Hello guys, 
Sorry if this was supposed to posted on the wrong area. This is my first attempt to question and making the web through the Fronted Mentor. 
I customized the font but when I uploaded the file, the font I set has gone. 
I read bunch of similar questions and tried on my own but couldn't figure out. 

Would anyone please help me out? 

my repository is[ here](https://github.com/Uppi-Sempai/web.git)

and my web looks is [here](https://uppi-sempai.github.io/web/)

Thank you very much!!

**Comment by Uppi-Sempai** [comment]
Sorry Not sure why the question became "body" ><;

**Comment by Berchez** [comment, marked as answer]
Here are a few steps you can follow to troubleshoot this issue:

1. Check Font Path: Ensure that the path to your custom font files is correct. If the path is incorrect, the browser won’t be able to find and load the font.

2. Use @font-face Rule: Make sure you’re using the @font-face rule correctly in your CSS. Here’s an example:

`@font-face {
  font-family: 'MyCustomFont';
  src: url('./fonts/MyCustomFont.woff2') format('woff2'),
       url('./fonts/MyCustomFont.woff') format('woff');
  font-weight: normal;
  font-style: normal;
}`

3. Check File Formats: Ensure that you have included all necessary font formats (e.g., WOFF, WOFF2, TTF) to support different browsers.

4. Clear Cache: Sometimes, browsers cache old versions of your files. Clear your browser cache or use a private/incognito window to see if the font loads correctly.

5. Verify Deployment: Double-check that your font files are correctly uploaded to the server and accessible. You can do this by navigating directly to the font file URL in your browser.

6. Console Errors: Check the browser console for any errors related to loading the font files. This can give you clues about what’s going wrong.

**Reply by Uppi-Sempai** [reply]
Thank you very much for your response and advice. I'm gonna work on it now to see ! :)

**Reply by Uppi-Sempai** [reply]
Omg it took 35 mins  to figure out lol 
Thank you so much, it worked. I did not include the format as "TTF". I learnt so much from you, thank you!
